src/main/java/core/basesyntax/ConsoleHandler.java

```diff
@@ -0,0 +1,174 @@
+                .flatMap(o -> o)
+                .filter(str -> str.matches("^[a-zA-Z]+"))
+                .count();
+    }
+
+    private void help(String[] args) {
+        if (args.length >= 2) {
+            System.out.println(commandInfo.get(args[1]));
+            return;
+        }
+        for (String info : commandInfo.values()) {
+            System.out.println(info);
+
+        }
+
+    }
+
+    private void saveText(String input) throws IOException {
+        if (input.isEmpty()) {
+            return;
+        }
+        System.out.println("Save text? Y/N");
+        if (askYesNo()) {
+            String[] filePath = askUserFilePath();
+            create(filePath[0], filePath[1]);
+            writeToFile(filePath[0], filePath[1], input);
+        }
+    }
+
+    private boolean askYesNo() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String answer = scanner.next();
+            switch (answer) {
+                case "Y":
+                    return true;
+                case "N":
+                    return false;
+                default:
+                    System.out.println("Unknown answer. Please, write Y or N");
+            }
+        }
+    }
+
+    private void writeToFile(String path, String fileName, String content)
+            throws FileNotFoundException {
+        try (PrintWriter writer = new PrintWriter(path + File.separator + fileName)) {
+            writer.write(content);
+        }
+    }
+
+    private String[] askUserFilePath() {
+        System.out.println("Please, write path and name for the new text file:");
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.nextLine();
+        return input.split(" ");
+    }
+
+    private void read(String path, String fileName) {
+        File file = new File(path + File.separator + fileName);
+        if (!file.exists()) {
+            System.out.println("Warning: no such file or directory");
+            return;
+        }
+        try (BufferedReader reader = new BufferedReader(
+                new FileReader(path + File.separator + fileName))) {
+            String line = reader.readLine();
+            while (line != null) {
+                System.out.println(line);
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private void create(String pathName, String fileName) {
+        try {
+            File directory = new File(pathName);
+            if (!directory.exists()) {
+                System.out.println("Warning: no such directory");
+                return;
+            }
+            File file = new File(pathName + File.separator + fileName);
+            if (file.exists()) {
+                System.out.println("File " + fileName + " at " + pathName + " already exists.\n"
+                        + "Rewrite? Y/N");
+                if (askYesNo()) {
```

Comment:
if user enters no, we'll call file.createNewFile(). Is that expected?

Reply:
@roman-mate createNewFile method does not rewrite a file, if it exists, so we should delete file first. So this works well :)